src/clover/optimisation/appraisal.py

```diff
@@ -237,7 +248,69 @@ def _simulation_environmental_appraisal(  # pylint: disable=too-many-locals
     )
 
 
+# Voltage:
+#   The voltage rate of the location being considered.
+VOLTAGE: int = 220
+
+#move to finance
+def _get_grid_pricing_tier(
+    grid: pd.DataFrame,
+    grid_energy: pd.Series,
+    grid_tier: GridTier,
+    household_monthly_demand=pd.Series, 
+    #  """
+    # for years in lifetime (20 years)
+    #   for months in year (12 months)
+    #       for days in months (30 days)
+    #           monthly_demand=sum(grid_energy.values())
+    #  """
+): 
+    """
+    Gets the grid pricing tier.
+
+        Inputs:
+            - grid:
+                Needed to identify the grid criteria and data.
+            - grid energy:
+                Energy that was supplied by the grid.
+            - Daily household hourly consumption:
+                Needed to identify the peak consumption per day.
+            - Monthly household consumption:
+                Needed to identify the consumption tier of a household on a monthly basis.
+        Outputs:
+            - tier:
+                The tier corresponding to the household consumption based on the grid in use.
+    """
+
+    for grid_type in grid:  # run over the list of tiers where we have the different upper bound consumption
```

Comment:
@BenWinchester LATEST UPDATE I think getting closer but Ben I seriously don't know how to write this. I've been stuck on it days and days. Please help. I think the methodology is very clear and written I just don't know exactly how does the syntax work.